using-timescaledb/continuous-aggregates.md

```diff
@@ -139,19 +148,60 @@ PostgreSQL. In addition, TimescaleDB continuous aggregates does not
 currently support the `FILTER` clause even though it is possible to
 parallelize but we might add support for this in a future version.
 
+By default, a continuous aggregate is refreshed when created, which
+means that it is immediately populated with data. Since this can be a
+lengthy process it is sometimes desirable to defer materialization to
+a manual refresh or a policy that does it in the background. You can
+disable the immediate materialization of data using the `WITH NO DATA`
+option to `CREATE MATERIALIZED VIEW`.
+
+#### Automatic refreshing with a continuous aggregate policy
+
+You can refresh the continuous aggregate view manually as above, but
+it is also possible to automate the refreshing by adding a continuous aggregate 
+policy with [`add_continuous_aggregate_policy`][add-continuous-aggregate-policy]. 
+For example:
+
+```sql
+SELECT add_continuous_aggregate_policy('device_summary',
+	start_interval => NULL,
+	end_interval => INTERVAL '2 h',
+	schedule_interval => INTERVAL '2 h');
+```
+
+When the policy runs it will refresh a window of data 
+that is given by a start and end interval, which are relative to the
+[`CLOCK_TIMESTAMP`][clock_timestamp] at the time of executing the policy. Similar to the
+`refresh_continuous_aggregate` function, providing `NULL` to either of
+the intervals will always refresh the entire continuous aggregate.
+
+The `schedule_interval` determines how often the policy job runs, so
+the most recent value in the continuous aggregate lags behind with at
```

Comment:
I think this is confusing. Would suggest you drop the latter half of the sentence and restate as 
```suggestion
The schedule_interval determines how often the policy job runs.
```
An example that modifies the schedule_interval using alter_job would be pertinent here.